### DISCUSSION TITLE
Change the blinds to be displayed as switches

### DISCUSSION THREAD

**Discussion by OmriMulti** [aarons22/homebridge-bond#236] [Q&A]
I Have Somfy Blinds and I was wondering if there is a configuration to change the blinds to be displayed as switches instead of blinds in Homebridge. right now, If i'm using the blinds remote control to close them. then its displayed as open on the Homebridge app. 
I just want the option to have 3 switched like in the bond app. Open, Close, My.

**Comment by rbpasker** [comment]
agreed. The Somfy remotes have momentary press switches, and should show up as momentary press switches
